test/test_datasets.py

```diff
@@ -595,6 +595,18 @@ def test_attr_names(self):
         with self.create_dataset() as (dataset, info):
             assert tuple(dataset.attr_names) == info["attr_names"]
 
+    def test_images_names_split(self):
+        splits = ["train", "valid", "test"]
+        with self.create_dataset(split='all') as (dataset, _):
+            all_imgs_names = dataset.filename
+
+        merged_imgs_names = []
+        for split in splits:
+            with self.create_dataset(split=split) as (dataset, _):
+                merged_imgs_names += dataset.filename
+
+        assert(merged_imgs_names == all_imgs_names)
```

Comment:
`assert` is a statement, so we don't need the parentheses:

```suggestion
        assert merged_imgs_names == all_imgs_names
```

Reply:
Agreed